### ISSUE TITLE
cbuildgen: RTE/_<build-type>_<target-type> 

### ISSUE THREAD

**Issue by jkrech** [Open-CMSIS-Pack/devtools#882]
**Describe the bug**
A csolution project with <target-type> but no <build-type> will be converted by the `csolution tool` into a file named:
`<projectname>+<target-type>.cprj` and will generate a folder `RTE/_<target-type>` containing e.g. RTE_Components.h

`cbuildgen tool` by default does not modify the content of directories in `RTE/...`.
In this case `cbuildgen` throws an error:

```
*** ERROR M204:
  Path not found: '...<projectdir>/RTE/_Target_1'!
```

**To Reproduce**
Take a csolution project and remove/disable <build-type>.

**Expected behavior**
The expected behavior of cbuildgen is that it recognizes the <projectname>+<target-type>.cprj as name of the <target-type> and expects the `<projectdir>/RTE/_<target-type>` instead of `<projectdir>/RTE/_Target_1`.
It seems like `+` is not recognized as `separator` between <projectname> and <target-type>. Once I replace `+` with `.` in the cprj filename the correct RTE/_<target-type> directory is used.

**Environment (please complete the following information):**

- Tool: cmsis-toolbox/cbuildgen
- Version: 1.6.0
- OS: windows

**Comment by jkrech** [comment]
Current workarounds:
a) add a build-type to your *.csolution.yml file:
b) invoke cbuild with --update-rte to create content into `_Target_1`

**Comment by brondani** [comment]
Fix released in cmsis-toolbox [1.7.0](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/releases/tag/[1.7.0](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/releases/tag/1.7.0))
